pkg/util/override.go

```diff
@@ -1,10 +1,9 @@
-package validation
+package util
```

Comment:
Two reasons:
1) it has nothing to do with validation, and is more generic type. Specifically, I wanted to avoid any references to Limit from OverridesManager.
2) Since this changes functionality of this type, I wanted to break existing client code, to make sure it is updated.

Reply:
Now it lives in pkg/util/runtime_config/manager.go (as runtime_config.Manager type).